lib/constants/page.ts

```diff
@@ -0,0 +1,9 @@
+export const ArticlePageSize = 4;
+export const ProductsPageSize = 4;
+
+export const notFoundRedirect = {
```

Comment:
Use the https://nextjs.org/docs/pages/api-reference/functions/get-static-props#notfound redirect directly, i would like to avoid situation we change the notFound behavior and we need to check all of the usage to decide whether to change it or not.